### ISSUE TITLE
Add images: Android Basics with Compose

### ISSUE THREAD

**Issue by GregKanczes** [google-developer-training/basic-android-kotlin-compose-birthday-card-app#455]
**URL of codelab**
https://developer.android.com/courses/pathways/android-basics-compose-unit-1-pathway-3

**In which task and step of the codelab can this issue be found?**
task = 4
step = 3 - Add an image composable

**Describe the problem**

val image = painterResource(R.drawable.androidparty)

Unresolved reference: androidparty

**Steps to reproduce?**
1. Import androidparty.png to res/drawable-nodpi 
2. Add composable function:

@Composable
fun GreetingImage(message: String, from: String, modifier: Modifier = Modifier) {
    val image = painterResource(R.drawable.androidparty)
    Image(
        painter = image,
        contentDescription = null
    )
}

3. See error "Unresolved reference: androidparty" and no reference in the Resource Manager Drawable section

**Versions**
_Android Studio version:_ Android Studio Iguana | 2023.2.1 Canary 7
_API version of the emulator:_ 

**Additional information**

![android app error - androidparty png import - 20231018](https://github.com/google-developer-training/basic-android-kotlin-compose-birthday-card-app/assets/11502806/78e1ee30-5835-4a4a-80ff-d6f31343e6e4)
![android app error - androidparty png import - resource manager view - 20231018](https://github.com/google-developer-training/basic-android-kotlin-compose-birthday-card-app/assets/11502806/32be70af-7d33-4d3e-9658-ad15c617cde4)

**Comment by johnshea** [comment]
Hi @GregKanczes 

Thank you for writing in with your issue.

I noticed that you are running a canary version of Android Studio. While it should work, there are no guarantees.

If you are able and have time, please try the codelab again on a stable version of Android Studio. Currently this is Giraffe.

This is to rule out that the canary version is causing the issue.

Hope you are enjoying the course!

Regards.

**Comment by GregKanczes** [comment]
It works on Giraffe.  Thanks.

-Greg

**Comment by johnshea** [comment]
Hi @GregKanczes 

Quick question - did you just load this "broken" code in Giraffe and it ran or did you retype the codelab from scratch?

Trying to narrow down if it might be a typing issue.

Regards.

**Comment by GregKanczes** [comment]
I opened the same project in Giraffe and imported the image into the
resource manager again.  It imported properly, I was able to see it in the
resource manager and reference it in the code.  I then switched the
androidparty.png with a pic of my sister and me, updated the text, and sent
a screenshot of the app from my phone to my sister  for her birthday
yesterday.

I was very happy and so was she when she got it!

Thanks!
